redis/vibe/db/redis/redis.d

```diff
@@ -130,6 +130,20 @@ final class RedisClient {
 		});
 	}
 
+	/** Release all connections that are not in use. Call before exiting the
+	 * program to avoid relying on the GC to clean up the sockets.
+	 */
+	void releaseUnusedConnections() @safe
+	{
+		// the try/catch in this is for the old vibe.d:core library, not
+		// necessary in vibe-core, but we have to work with both.
+		m_connections.removeUnused((conn) @safe nothrow {
+			try {
+                 conn.m_conn.close();
+			 } catch(Exception e) {}
```

Comment:
```suggestion
			 } catch(Exception e) logDebug("Failed to close unused Redis connection: %s", e.msg);
```

Not sure whether this can actually occur in the implementation, but at least that way there is a chance to notice unexpected behavior.

Reply:
Admittedly, this was simply to shut up the compiler. It only requires this for the `vibe-d:core` version, as the interface does not require `nothrow` on the `close` method.

I will update to log the exception as requested.